test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1324,6 +1324,31 @@ UCS_TEST_P(select_transport_rndv, select_dc, "NUM_EPS=300")
 
 UCP_INSTANTIATE_TEST_CASE_TLS(select_transport_rndv, ib, "ib")
 
+class select_transport_rma_bw : public test_ucp_wireup {
+public:
+    static void get_test_variants(std::vector<ucp_test_variant> &variants)
+    {
+        add_variant_with_value(variants, UCP_FEATURE_RMA, TEST_RMA, "rma");
+    }
+};
+
+UCS_TEST_P(select_transport_rma_bw, select_rc, "NUM_EPS=56", "NUM_PPN=28")
+{
+    sender().connect(&receiver(), get_ep_params());
+    const ucp_ep_config_t *config = ucp_ep_config(sender().ep());
+
+    for (int i = 0; (i < config->key.num_lanes) &&
+                    (config->key.rma_bw_lanes[i] != UCP_NULL_LANE);
+         ++i) {
+        /* Verify RC is selected for rma_bw lanes */
+        ucp_lane_index_t lane = config->key.rma_bw_lanes[i];
+        ASSERT_STREQ("rc_mlx5",
+                     ucp_ep_get_tl_rsc(sender().ep(), lane)->tl_name);
+    }
+}
+
+UCP_INSTANTIATE_TEST_CASE_TLS(select_transport_rma_bw, tcp_ib, "tcp,ib")
```

Comment:
does it work if there is no IB device on the machine?

Reply:
how can I check that? is there any way to simulate it? or do I need a machine with no IB devices?